gwctl/cmd/describe.go

```diff
@@ -121,7 +146,15 @@ func runDescribe(cmd *cobra.Command, args []string, params *utils.CmdParams) {
 		gwPrinter.PrintDescribeView(resourceModel)
 
 	case "gatewayclass", "gatewayclasses":
-		filter := resourcediscovery.Filter{}
+		selector, err := labels.Parse(labelSelector)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Unable to find resources that match the label selector \"%s\": %v\n", labelSelector, err)
+			os.Exit(1)
+		}
+		filter := resourcediscovery.Filter{
+			Namespace: ns,
+			Labels:    selector,
+		}
```

Comment:
```suggestion
		filter := resourcediscovery.Filter{
			Labels:    selector,
		}
```
as GatewayClass is a cluster-scoped object